java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -4959,10 +4909,13 @@ public void actionPerformed(ActionEvent e) {
                 (inColor != null) && (inColor == turnoutCircleColor), a);
     } //addTurnoutCircleColorMenuEntry
 
-    private void addTurnoutCircleSizeMenuEntry(@Nonnull JMenu inMenu, @Nonnull String inName, final int inSize) {
+    private void addTurnoutCircleSizeMenuEntry(
+            @Nonnull JMenu inMenu,
+            @Nonnull String inName,
+            final int inSize) {
         ActionListener a = new ActionListener() {
             @Override
-            public void actionPerformed(ActionEvent e) {
+            public void actionPerformed(@Nonnull ActionEvent e) {
```

Comment:
ActionListeners in the Java source code accepts null as an ActionEvent, so I am not sure that AWT or Swing components may not also possibly send a null ActionEvent that may be passed here. Since you don't use ```e``` in this method, you also don't care that it is not null.

Reply:
So… should I change it to @Nullable? or just remove it? If someone adds code there later it would be good for FindBugs to "do the right thing" (and verify that that code checks for null before using e).

Reply:
I would just remove the annotation. Findbugs is very good about finding potential NPEs, and will flag the potential NPE if it ever encounters it.

(Also, since we do not own the ActionListener API, we do not control it, so we should not indicate restrictions on it that may change as other third-party libraries use and implement the API.)

Reply:
Done…